app/src/main/java/com/eventyay/organizer/core/event/create/UpdateEventFragment.java

```diff
@@ -336,6 +352,19 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 binding.form.description.setText(description);
                 binding.form.description.setTextColor(Color.BLACK);
             }
+        } else if (requestCode == IMAGE_CHOOSER_REQUEST_CODE && resultCode == RESULT_OK) {
+            Uri selectedImageUri = data.getData();
+            try {
+                InputStream imageStream = getActivity().getContentResolver().openInputStream(selectedImageUri);
+                Bitmap bitmap = BitmapFactory.decodeStream(imageStream);
+                String encodedImage = Utils.encodeImage(bitmap);
+                ImageData imageData = new ImageData("data:image/gif;base64,"+encodedImage);
```

Comment:
Why data/gif?

Reply:
It's defined like that in the API, isn't it?

Reply:
No, it's an example

Reply:
But when I tried without this, it didn't work

Reply:
You have to add proper meta, not just image/gif always

Reply:
Done